### PR TITLE
Use the new plugin-outlets in Discourse 1.3

### PR DESCRIPTION
**Pull request by nlalonde**
I added some plugin outlets to Discourse today for ads from this plugin.

https://github.com/discourse/discourse/commit/ccce27d1b81486f5d92261babeb1b5227f8592dd

This pull request updates this plugin so it doesn't overwrite any templates, but depends on my commit! So you probably don't want to merge it until 1.3 stable is released.

### CODE REVIEW

File: plugin.rb [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+# version: 1.3.0
 # author: DiscourseHosting.com
 # url: https://www.github.com/discoursehosting/discourse-adsense
-# Supported Discourse version: v1.2
+# Supported Discourse version: v1.3
 
 register_asset "javascripts/adsense.js"
-register_asset "javascripts/discourse/templates/discovery.hbs"
-register_asset "javascripts/discourse/templates/topic.hbs"
+register_asset "javascripts/discourse/templates/connectors/discovery-list-container-top/discovery-list-container-top.hbs"
```
Comment (ZogStriP) [inline plugin.rb]: aren't connectors picked up automamigally?

**Comment by riking** [comment]
The container name (last part of the filename) should be "adsense".

**Comment by nlalonde** [comment]
The last part of which filename?

**Comment by nlalonde** [comment]
Good comments, thanks! I fix and submit pr again later.

**Pull request closed without merging**